docs/user/discover.asciidoc

```diff
@@ -3,7 +3,28 @@
 
 [partintro]
 --
-**_Tell {kib} where to find your data, then search and filter it for hidden insights and relationships._**
+**_Exploring and querying your data._**
```

Comment:
```suggestion
**_Explore and query your data._**
```

Reply:
wanted to align with the video:

![Bildschirmfoto 2021-03-03 um 12 05 10](https://user-images.githubusercontent.com/463851/109796695-c4ef2b80-7c18-11eb-8a43-b8192fe572f9.png)